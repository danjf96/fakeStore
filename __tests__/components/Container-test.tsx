/**
 * @format
 */

 import 'react-native';
 import React from 'react';
 import Container from '../../src/components/container';
 
 // Note: test renderer must be required after react-native.
import { Text } from 'react-native';
import { render } from 'react-native-testing-library';
 
describe('renders correctly', () => {
    it('basic', () => {
        const screen = render(<Container />);
        expect(screen.toJSON()).toMatchSnapshot()
    });

    it('with children', () => {
        const { getByText } = render(<Container >
            <Text>Teste</Text>
        </Container>);
        expect(getByText('Teste')).toBeTruthy()
    });
})

 