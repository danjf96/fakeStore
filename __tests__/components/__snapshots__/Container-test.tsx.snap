// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly basic 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "white",
          "padding": 19,
          "width": "100%",
        },
        undefined,
      ]
    }
  />
</RCTSafeAreaView>
`;
